app/models/project.rb

```diff
@@ -12,6 +12,10 @@ def self.find(id)
   end
 
   # -- Instance Methods -----------------------------------------------------
+  def authors
```

Comment:
also present in https://github.com/dradis/dradis-ce/pull/498#pullrequestreview-287401164